Dockerfile

```diff
@@ -33,11 +33,10 @@ RUN apt-get update \
   && DEBIAN_FRONTEND=noninteractive apt-get install --yes --quiet --no-install-recommends \
   sudo ca-certificates apt-transport-https wget locales openssh-client gpg gpg-agent \
   supervisor logrotate git gettext-base tzdata inotify-tools psmisc \
+  libldap-common gcc libsasl2-dev libldap2-dev \
```

Comment:
I prefer to keep Dockerfile as short as possible and move these staff to [`assets/build/install.sh`](assets/build/install.sh)

There are two variable defined there:

- `REQUIRED_PACKAGES` where you can add `libldap-common`
- `BUILD_DEPENDENCIES` where you can add `gcc`, `libsasl2-dev`, `libldap2-dev`.

The second variable is clean at the end of the script.

https://github.com/cdalvaro/docker-salt-master/blob/44c73c718b652b94b180c9214acdb9c8417e04b1/assets/build/install.sh#L13-L17

Reply:
Oops - I missed that most of the work can happen in install.sh - I'll move the relevant bits over to it. I think that will mean that there wiull be no changes in `Dockerfile` in this PR at the end, so let me know if you're happy for me to rebase the last commit to it away, or would prefer an explicit 'revert' in the commit history.

Reply:
I'm fine rebasing!